packages/kit/src/exports/vite/index.js

```diff
@@ -554,12 +547,69 @@ function kit({ svelte_config }) {
 				fs.unlinkSync(`${paths.output_dir}/client/${vite_config.build.manifest}`);
 				fs.unlinkSync(`${paths.output_dir}/server/${vite_config.build.manifest}`);
 			}
+		},
+
+		/**
+		 * @see https://vitejs.dev/guide/api-plugin.html#configureserver
+		 */
+		async configureServer(vite) {
+			// set `import { version } from '$app/environment'`
+			(await vite.ssrLoadModule(`${runtime_prefix}/env.js`)).set_version(
+				svelte_config.kit.version.name
+			);
+
+			const paths = await vite.ssrLoadModule(`${runtime_base}/paths.js`);
+			const assets = svelte_config.kit.paths.assets
+				? SVELTE_KIT_ASSETS
+				: svelte_config.kit.paths.base;
+			paths.set_paths({
+				base: svelte_config.kit.paths.base,
+				assets
+			});
 		}
 	};
 
 	/** @type {import('vite').Plugin} */
-	const plugin_middleware = {
-		name: 'vite-plugin-sveltekit-middleware',
+	const plugin_build = {
+		name: 'vite-plugin-sveltekit-build',
```

Comment:
this naming feels kinda confusing to me — this applies in dev too, no?

Reply:
yeah. renamed it to bundle. maybe that's better?

Reply:
not really — no bundling happens in `dev` — though i don't have a better suggestion off the top of my head just yet

Reply:
some stuff gets bundled (e.g. stuff in `node_modules`), but yeah I guess not everything does. how about `compile`? all the `.svelte` files are compiled even in dev. I just pushed a change to rename it

Reply:
yeah, seems fine